.ci/travis_checks.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# guarantee script exits with non-zero code
+# the first time any of these commands fail
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
What is this doing?

Reply:
I just added a comment. It gets the directory of the file, rather than the working directory, so that you can run it from anywhere.

Reply:
nice!